common/src/main/java/io/pravega/common/util/ServiceConfig.java

```diff
@@ -36,8 +32,8 @@
     public static final Property<Integer> ZK_RETRY_COUNT = Property.named("zkRetryCount", 5);
     public static final Property<Integer> ZK_SESSION_TIMEOUT_MS = Property.named("zkSessionTimeoutMs", 10000);
     public static final Property<String> CLUSTER_NAME = Property.named("clusterName", "pravega-cluster");
-    public static final Property<String> STORAGE_IMPLEMENTATION = Property.named("storageImplementation",
-                                                        "INMEMORY");
+    public static final Property<String> STORAGE_FACTORY_IMPLEMENTATION_CLASS = Property.named("storageImplementation",
```

Comment:
config.properties was not updated with this change.

Ideally the property should indicate that it is a classname. Perhaps `storageImplementationClass` would be a better name.